bin/storybookGenerator.ts

```diff
@@ -1,12 +1,11 @@
 import * as fs from "fs"
 import * as path from "path"
-
-import { componentsConfig, documentComponents } from "./muiComponentConfig"
+import { componentsConfig, documentComponents } from "./muiComponentConfig.tsx"
 import {
   componentTemplate,
   storiesJustArgsTemplate,
   storiesTemplate,
-} from "./templates"
+} from "./templates.ts"
```

Comment:
is it required the `.ts`?

Reply:
yes it's because we are executing `node --loader ts-node/esm ./bin/storybookGenerator.ts`